src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    private int supply = 0;
+    private int buy = 0;
```

Comment:
The variables 'supply' and 'buy' should be local variables inside the readFile method rather than class fields, as they do not represent the state of the class but are rather used for counting within a method.